### PR TITLE
Resume connection stream before continuing.

### LINKED ISSUE #80: Websocket stuck after receiving around ~16 kb of data from client
## 🐛 Bug Report

After client sends ~100 messages of ~160 bytes each to websocket, `connection.socket.on('message')` is no longer being triggered.

It seems to be size-dependent, I can send thousands of very short (a few bytes) messages, but only around 100 lines of 160 bytes.

## To Reproduce

In new dir:

```
yarn init
yarn add fastify fastify-websocket
```

Create `server.js`:

```
'use strict'

const fastify = require('fastify')()

fastify.register(require('fastify-websocket'))

fastify.get('/', { websocket: true }, (connection, req) => {
  connection.socket.on('message', message => {
    // message === 'hi from client'
    connection.socket.send('hi from server')
  })
})

fastify.listen(5000, err => {
  if (err) {
    fastify.log.error(err)
    process.exit(1)
  }
```

Run `node server.js`

In another window, run:
```
wscat --show-ping-pong -c "ws://localhost:5000"
```

Create a file with ~150 lines of 160 bytes each. Copy-paste contents into wscat.

The server will stop responding.

## Expected behavior

Should keep returning responses.

## Your Environment

- *node version*: 12
- *fastify version*: >=2.0.0
- *os*: Mac, Linux

### PR DESCRIPTION
**Pull request by ShogunPanda**
Fixes #80.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -97,6 +97,13 @@ function fastifyWebsocket (fastify, opts, next) {
     const response = new ServerResponse(request)
     request[kWs] = WebSocket.createWebSocketStream(connection)
     request[kWs].socket = connection
+
+    request[kWs].socket.on('newListener', event => {
+      if (event === 'message') {
+        request[kWs].resume()
+      }
+    })
```
Comment (mcollina) [inline index.js:105]: I think you can remove https://github.com/fastify/fastify-websocket/pull/81/files#diff-168726dbe96b3ce427e7fedce31bb0bcR83-R87 now.

**Review by mcollina — COMMENTED** [review]
just a note

File: index.js (hunk 1 above)
Comment (ShogunPanda) [reply index.js:99]: Done

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**